Tasks/DockerV2/Tests/L0.ts

```diff
@@ -553,6 +553,21 @@ describe("DockerV2 Suite", function () {
         done();
     });
 
+    it('Docker start should start unknown container', (done:MochaDone) => {
```

Comment:
This is a bit confusing. How would it start an unknown container?

Reply:
I'll change to "unregistered". The idea is that the agent doesn't know about the container